src/test/java/com/luxoft/training/jva008/essensial/PropertiesTutor.java

```diff
@@ -31,7 +34,7 @@ public void testJavaVersion() {
      */
     public Properties getProperties() {
         Properties prop = new Properties();
-        try (InputStream is = new FileInputStream("files/props.properties")) {
+        try (InputStream is = new FileInputStream("C:\\Users\\qtr-ptc-69\\IdeaProjects\\new\\jva-008\\src\\test\\resources\\props.properties")) {
```

Comment:
@bewuer please do not change this line and make it working
```suggestion
        try (InputStream is = new FileInputStream("files/props.properties")) {
```

Reply:
new path added